controllers/pullRequests.js

```diff
@@ -38,7 +38,7 @@ const getUserPRs = async (req, res) => {
 const getStalePRs = async (req, res) => {
   try {
     const { size, page } = req.query;
-    const { data } = await githubService.fetchStalePRs(size, page);
+    const { data } = await githubService.fetchOpenPRs("asc", size, page);
```

Comment:
Should this come from constants? (Or better, using TypeScript)

Reply:
NIT: `fetchOpenPRs(page, size, SORTING.ASC);`

Reply:
> Should this come from constants? (Or better, using TypeScript)

@ankushdharkar not sure what do you mean by `using TypeScript`?